### PR TITLE
KOGITO-5933: Create new boxed expression editor sanity checks

### PR DESCRIPTION
**Pull request by jomarko**
Follow up PR for #631

### CODE REVIEW

**Comment by jomarko** [comment]
VS Code tests needs to finish and cleanup

**Comment by jomarko** [comment]
VS Code tests finished.

**Review by jstastny-cz — APPROVED** [review]
The kie-editors-standalone tests look good. Thank you.

File: packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors.test.ts [hunk 1]
```diff
@@ -113,7 +114,7 @@ describe("Editors are loading properly", () => {
     await webview.switchBack();
   });
 
-  it("Undo command in DMN Editor", async function () {
+  it.skip("Undo command in DMN Editor", async function () {
```
Comment (domhanak) [inline packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors.test.ts]: I guess you ll remove these skips, so just putting a reminder.

**Review by domhanak — APPROVED** [review]
Looks good thanks

**Review by vpellegrino — APPROVED** [review]
Fine from my point of view. Thank you!

File: packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors.test.ts (hunk 1 above)
Comment (jomarko) [reply packages/vscode-extension-pack-kogito-kie-editors/it-tests/extension-editors.test.ts]: yes, should be reverted now, thank you @domhanak

**Comment by jomarko** [comment]
added also chrome-extension sanity check, PR should be ready for ready for review on that channel and possibly merge.

**Review by yesamer — APPROVED** [review]

**Comment by jomarko** [comment]
@tomasdavidorg may I ask you to review chrome extension part? no pressure, whenever you have a moment.

**Review by tomasdavidorg — APPROVED** [review]
Chrome extension part looks good to me. Thanks

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge

**Comment by ederign** [comment]
@triceo next time, could you please wait for either mine or Tiago's review? Thanks.

**Comment by jomarko** [comment]
@ederign please provide your comments, I can address them retrospectively.

**Comment by ederign** [comment]
@jomarko there is nothing specific with this PR, but please before merging anything on the kogito-tooling repo, please reach out to either Tiago or me.

**Comment by tarilabs** [comment]
we should reserve pinging `@kiegroup/gatekeepers` only in those situations where an SME with Merge rights is not reachable directly.
I receive these notifications too, and if I see all CI are green, what would prevent me to click the Merge button?
That's why I would suggest, maybe it's best to use gatekeepers wide distribution list only for those situations like ~"everyone on my team is on PTO, reaching out for someone with merge rights".

very imho and very my2c :)

**Comment by jomarko** [comment]
or readme/contribution guide, with names to contact within given repository?

**Pull request merged**